.npmignore

```diff
@@ -1,2 +1,8 @@
 node_modules
-npm-debug.log
\ No newline at end of file
+npm-debug.log
+src
```

Comment:
i'd prefer to leave this one, find myself looking at src in node_modules directory quite often